docker/experimental/ansible/config/vars.yaml

```diff
@@ -1,7 +1,10 @@
 # Used for fetching cuda from the right repo, see apt.yaml.
 cuda_repo: ubuntu1804
+cuda_version: 11.8
 # Used for fetching clang from the right repo, see apt.yaml.
 llvm_debian_repo: buster
-clang_version: 11
+clang_version: 10
```

Comment:
It seems that we're going for gcc in the end until clang/bazel interaction is fixed. We should keep clang around though for other things so nothing to change here, just fyi.

Reply:
Ack